src/components/v2/V2Project/V2ProjectReconfigureModal/index.tsx

```diff
@@ -216,8 +216,13 @@ export default function V2ProjectReconfigureModal({
     effectiveDistributionLimit = queuedDistributionLimit
   }
 
-  const effectiveDistributionLimitCurrency =
-    queuedDistributionLimitCurrency ?? distributionLimitCurrency
+  let effectiveDistributionLimitCurrency = distributionLimitCurrency
```

Comment:
Fixes bug where distributionLimitCurrency is 0 (so defaults to ETH) when funding duration is unset (queuedFundingCycle doesn't exist)